### PR TITLE
Fix max transfer amount

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Add regex to validate transfer amount

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1670

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Comment by AlaaElattar** [comment]
- I have question, the fee excluded from the transfer amount or from my balance ?
image

**Comment by zaelgohary** [comment]
@AlaaElattar I think the transaction (extrinsic) fee is deducted from the user who did it,

**Review by AlaaElattar — APPROVED** [review]

File: packages/playground/src/dashboard/transfer_view.vue [hunk 1]
```diff
@@ -37,8 +37,11 @@
                 :rules="[
                   validators.required('Transfer amount is required '),
                   validators.isNumeric('Amount should be a number.'),
-                  validators.min('Amount must be greater than 0', 0.00000000001),
-                  validators.max('Insufficient funds', freeBalance),
+                  validators.min('Amount must be greater than 0.', 0.001),
+                  validators.max('Insufficient funds.', freeBalance),
+                  validators.pattern('Amount can have 3 decimals only.', {
+                    pattern: /^\d+(\.\d{1,3})?$/,
```
Comment (AhmedHanafy725) [inline packages/playground/src/dashboard/transfer_view.vue:43]: I think we can use `isDecimal` instead of pattern

Comment (zaelgohary) [reply packages/playground/src/dashboard/transfer_view.vue]: Fixed.

**Review by AhmedHanafy725 — APPROVED** [review]

**Review by maayarosama — APPROVED** [review]

**Pull request merged**